### PR TITLE
Test for update tags property

### PR DESCRIPTION
**Pull request by maririos**
Tests changing, adding and deleting tags from setting

### CODE REVIEW

File: Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs [hunk 1]
```diff
@@ -348,6 +348,50 @@ public async Task UpdateWrongETag()
...
+            {
+                // Different tags
+                var testSettingDiff = responseGet.Result.Clone();
+                var settingTags = testSettingDiff.Tags;
+                if (settingTags.ContainsKey("tag1")) settingTags["tag1"] = "value-updated";
+                settingTags.Add("tag3", "test_value3");
+                testSettingDiff.Tags = settingTags;
+                
+                Response<ConfigurationSetting> response = await service.UpdateAsync(testSettingDiff, filter, CancellationToken.None);
+                ConfigurationSetting responseSetting = response.Result;
```
Comment (KrzysztofCwalina) [inline Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs]: I know it's not very discoverable, but you can do:
```c#
ConfigurationSetting responseSetting = await service.UpdateAsync(...);
```

File: Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs [hunk 2]
```diff
@@ -348,6 +348,50 @@ public async Task UpdateWrongETag()
...
 
+        [Test]
+        public async Task UpdateTags()
+        {
+            var connectionString = Environment.GetEnvironmentVariable("AZ_CONFIG_CONNECTION");
+            Assert.NotNull(connectionString, "Set AZ_CONFIG_CONNECTION environment variable to the connection string");
+            var service = new ConfigurationClient(connectionString);
+
+            Response<ConfigurationSetting> responseSet = await service.SetAsync(s_testSetting, CancellationToken.None);
+            Response<ConfigurationSetting> responseGet = await service.GetAsync(s_testSetting.Key, s_testSetting.Label, CancellationToken.None);
```
Comment (KrzysztofCwalina) [inline Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs]: If you don't need the return values, you can do simply:
```c#
await service.SetAsync(s_testSetting, CancellationToken.None);
```

File: Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs (hunk 1 above)
Comment (maririos) [reply Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs]: ha! I didn't know that. Thank you :)

File: Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs (hunk 2 above)
Comment (maririos) [reply Configuration/Azure.Configuration.Test/ConfigurationLiveTests.cs]: I didn't need the `responseSet` though, so deleted!

**Pull request merged**